aim/config.py

```diff
@@ -32,18 +32,6 @@
 
 LOG = logging.getLogger(__name__)
 
-global_opts = [
```

Comment:
I was mentioning [1] and [2]

[3] seems to be taking care of [1], though this patch needs work to address review comments. You still need a patch for [2]

[1] http://git.openstack.org/cgit/openstack/group-based-policy/tree/gbpservice/neutron/plugins/ml2plus/drivers/apic_aim/mechanism_driver.py#n107
[2] http://git.openstack.org/cgit/openstack/group-based-policy/tree/gbpservice/neutron/tests/unit/plugins/ml2plus/test_apic_aim.py#n96
[3] https://review.openstack.org/#/c/412022/